kobo/apps/openrosa/libs/utils/logger_tools.py

```diff
@@ -546,6 +547,8 @@ def safe_create_instance(username, xml_file, media_files, uuid, request):
         error = OpenRosaResponseNotAllowed(t("Form is not active"))
     except TemporarilyUnavailableError:
         error = OpenRosaTemporarilyUnavailable(t("Temporarily unavailable"))
+    except AccountInactiveError:
+        error = OpenRosaResponseNotAllowed(t('Account is not active'))
```

Comment:
Seems kind of weird to return HTTP 405 for this, but I'm following the behavior of inactive (archived) forms as shown above